### ISSUE TITLE
Привет вот так выходит юзаю сокс5 помоги пожалуйста в чем может быть дело?

### ISSUE THREAD

**Issue by Karimadze** [petyasoft/Blum#49]
![image](https://github.com/user-attachments/assets/92e53cce-28d4-4c2b-b0f5-6ead74ec4e4a)

**Comment by petyasoft** [comment]
может у сокс5 другой порт надо было указать? и попробуй хттп

**Comment by Karimadze** [comment]
Прокси рабочие сами по себе, в других сборках пашут, и в проксифаер проверял все ок, возможно ли что нужно в настройках что то поменять?

**Comment by petyasoft** [comment]
да, попробуй выбрать тип прокси в конфиге хттп и перепроверь порты у прокси
